src/components/Calendar/ClassModalContainer.js

```diff
@@ -0,0 +1,24 @@
+import { connect } from 'react-redux';
+import toJS from 'util/to-js';
+import { removeSection } from 'actions';
+import ClassModal from './ClassModal';
+
+/* istanbul ignore next */
+function mapStateToProps(state, ownProps) {
+  return {
+    section: ownProps.section,
```

Comment:
All of the props that are passed into the container are also passed into the child, so there's no need to pass it down yourself.